sys/log/full/src/log_nmgr.c

```diff
@@ -148,6 +150,9 @@ log_nmgr_encode_entry(struct log *log, struct log_offset *log_offset,
         return MGMT_ERR_ECORRUPT;
     }
 
+    g_err |= cbor_encode_text_stringz(&rsp, "flag");
+    g_err |= cbor_encode_uint(&rsp, ueh->ue_flag);
```

Comment:
Sorry for the late comment.  In my opinion, we should not include flags in the newtmgr response.  This just makes the response bigger without providing additional information.  The presence / absence of the image hash is what the client cares about, and it can determine that without seeing the flags.

Reply:
Got it. Removing it.